tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -8,10 +8,11 @@ details: |
   consistent behaviour across various distributions.
 
 environment:
-    DIRECTORY/ssl: /etc/ssl
+    DIRECTORY/apparmord: /etc/apparmor.d
 
 # This test only applies to classic systems
-systems: [-ubuntu-core-*]
+# fedora, centos, amazon: do not have /etc/apparmor.d
```

Comment:
No sure we we need to update this - AFAIK fedora/centos also have no /etc/ssl but the test for some reason did still work. Maybe the test needs some love and the change is fine but it seems it's not strictly needed (or am I missing something)?

Reply:
/etc/ssl exists in fedora/centos, that's why the test worked before afaict